misc/dbt-materialize/tests/adapter/test_basic.py

```diff
@@ -34,9 +34,29 @@
     run_dbt,
 )
 
+expected = {
+    "materialize_cloud": {
+        "base": "materializedview",
+        "view_model": "view",
+        "table_model": "materializedview",
+        "swappable": "materializedview",
+    },
+    "materialize_binary": {
```

Comment:
It's late and my brain is tired, but I'm not following why these are all expected to match `view` for the binary. Isn't `table` supposed to map to `materializedview`? Same for `base`; it's a seed model, so how does it not map to `materializedview`?

Reply:
Because the binary reports materialized views as type 'view'. 
```
materialize=> create materialized view t as select 1;
CREATE VIEW
materialize=> select * from mz_objects where id like 'u%';
 id |  oid  | schema_id |     name      | type
----+-------+-----------+---------------+-------
 u1 | 20288 |         3 | t             | view
 u2 | 20289 |         3 | t_primary_idx | index
(2 rows)
```

Reply:
Ah, right on!